src/components/editMeasure/measureDetails/measureInformation/MeasureInformation.test.tsx

```diff
@@ -140,7 +140,90 @@ describe("MeasureInformation component", () => {
   const { getByTestId, queryByText, findByTestId, getByRole, getByText } =
     screen;
 
+  it("Toast error shows when endorsing organization is not null and endorsement Id is empty", async () => {
+    measureStore.state.mockImplementationOnce(() => measure);
+    checkUserCanEdit.mockImplementationOnce(() => true);
+    await act(async () => {
+      render(<MeasureInformation setErrorMessage={setErrorMessage} />);
+      const endorserAutoComplete = await screen.findByTestId("endorser");
+      const endorserId = getByTestId(
+        "endorsement-number-input"
+      ) as HTMLInputElement;
+
+      fireEvent.keyDown(endorserAutoComplete, { key: "ArrowDown" });
+      // selects 2nd option
+      const endorserOptions = await screen.findAllByRole("option");
+      fireEvent.click(endorserOptions[1]);
+
+      // verifies if the option is selected
+      const endorserComboBox =
+        within(endorserAutoComplete).getByRole("combobox");
+      expect(endorserComboBox).toHaveValue("NQF");
```

Comment:
We'll eventually add more Endorsers. Consider modifying to assert non-null, non-empty string instead of specifically "NQF".

Reply:
Ignore me, just realized this is merging to the release branch.